### PR TITLE
Release v0.24.4

### PR DESCRIPTION
**Pull request by DmitriyMolch**

### CODE REVIEW

**Review by shamilkhan — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 ## vNext
 - ...
 
+## v0.24.4
```
Comment (AndreiNavoichyk) [inline CHANGELOG.md:6]: It should be `0.25.0`. It is not a hotfix. It is a new release.

Comment (DmitriyMolch) [reply CHANGELOG.md:6]: Actually, it was planned as a bug fix, not a hotfix or some new functionality. A hotfix is used when we need to fix something urgent in PROD and it's not the case. Here I did a bug fix for an old functionality but it's not urgent. This is why I updated the patch version.

Comment (AndreiNavoichyk) [reply CHANGELOG.md:6]: @DmitriyMolch 
Anything we are releasing from `dev` is new major or minor releases. Please, update the version to `0.25.0`

Comment (DmitriyMolch) [reply CHANGELOG.md:6]: In Jira this task was designed as a bug and not like a new feature. Only features can have minor version updates.

Comment (DmitriyMolch) [reply CHANGELOG.md:6]: ok

**Comment by DmitriyMolch** [comment]
Closed it as a new branch was created.

**Pull request closed without merging**